gradle/changelog/diff_command_content.yaml

```
@@ -0,0 +1,2 @@
+- type: fixed
+  description: Diff command content was always empty because the results were ignored ([#2036](https://github.com/scm-manager/scm-manager/pull/2036))
```

The reason is irrelevant for the changelog

```suggestion
  description: Diff command content ([#2036](https://github.com/scm-manager/scm-manager/pull/2036))
```